db/models/plan.js

```diff
@@ -1,11 +1,33 @@
-const Sequelize = require('sequelize')
-const db = require('../db')
+const Sequelize = require('sequelize');
+const db = require('../db');
 
-const ModelName = db.define('plan', {
+const Plan = db.define('plan', {
+  duration: {
+    type: Sequelize.INTEGER,
+    defaultValue : 1
+  },
+  end_date : {
+    type : Sequelize.DATE,
+    allowNull: true
+    },
+  },
+  {
+    instanceMethods : {
```

Comment:
// pretty sure this works

`countdown: function(plan) {
    let today = new Date()
    return plan.endDateCalc() - today 
}`